source/database-connections/mysql.txt

```diff
@@ -0,0 +1,295 @@
+.. _rm-connect-to-mysql:
+
+================
+Connect to MySQL
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. |db-type| replace:: MySQL
+
+.. include:: /includes/database-connections/connect-to-db-intro.rst
+
+.. _rm-mysql-connection-string:
+
+Steps
+-----
+
+To add a MySQL connection to Relational Migrator, you must:
+
+- Identify your :ref:`database connection string <rm-mysql-connection-string>`
+  or database information. 
+
+- :ref:`Create a service account. <rm-create-service-account-mysql>`
+
+- :ref:`Save a MySQL connection. <rm-connect-to-mysql-steps>`
+
+Database Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Relational Migrator requires a connection string in order to connect to your MySQL instance.
+You can provide the connection string directly, or provide information about your database
+which Relational Migrator uses to construct a connection string.
+
+The general form for a MySQL connection string is:
+
+.. code-block::
+
+   jdbc:mysql://<host:port>/<database>?<properties>
+
+For example, consider the following connection string:
+
+.. code-block::
+
+   jdbc:mysql://host1:3306/test
+
+The preceding connection string specifies these connection
+details:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Property
+     - Value
+   * - Host
+     - ``host1``
+   * - Port
+     - ``3306``
+   * - Database
+     - ``test``
```

Comment:
I'm in favor of cutting this entire thing, and having the line before the example say "For example, for database test running on host1, port 3306, the connections string is..."